tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/Record.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+
+namespace Azure.Sdk.Tools.TestProxy.Common
+{
+    public class RecordingHandlerSession
+    {
+        public RecordingHandlerSession() { }
```

Comment:
I don't think this is class is necessary or beneficial. I can simply add `Client` and `Path` to `ModifiableRecordSession`. There is no reason to add another layer of abstraction.